examples/linux_d555_dual_stream.py

```
@@ -0,0 +1,322 @@
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See README.md for detailed information.
+
+import argparse
+import ctypes
```

Flake8 (7.3.0)

[error] 'ctypes' imported but unused

(F401)